packages/amplify_authenticator/lib/amplify_authenticator.dart

```diff
@@ -425,21 +425,33 @@ class _AuthenticatorState extends State<Authenticator> {
     if (cognitoPlugin == null) {
       return const ['auth.plugins.Auth.Default'];
     }
+    if (cognitoPlugin.usernameAttributes == null) {
+      missingValues.add('\n - usernameAttributes');
+    }
     if (cognitoPlugin.signupAttributes == null) {
-      missingValues.add('auth.plugins.Auth.Default.signUpAttributes');
+      missingValues.add('\n - signUpAttributes');
     }
     if (cognitoPlugin.passwordProtectionSettings == null) {
-      missingValues.add('auth.plugins.Auth.Default.passwordProtectionSettings');
+      missingValues.add('\n - passwordProtectionSettings');
     }
     return missingValues;
   }
 
   @override
   Widget build(BuildContext context) {
     if (_configInitialized && _missingConfigValues.isNotEmpty) {
-      throw StateError('''Encountered problem(s) building the Authenticator.
-          Your amplifyconfiguration.dart file is missing required values: ${_missingConfigValues.join('\n')}). 
-          You should correct your amplifyconfiguration.dart file and restart your app.''');
+      throw StateError(
+        'Encountered problem(s) building the Authenticator.'
+        '\n\n'
+        'Your amplifyconfiguration.dart file is missing the following required attributes: ${_missingConfigValues.join()}'
+        '\n\n'
+        'This can occur if you are not using the Amplify CLI, or if the project was generated with the Amplify CLI prior to version X.Y.Z.' // TODO: Update version
```

Comment:
Can probably just use the version that was live when Authenticator was released?

Reply:
Yep, was 6.5.0 I believe, when zero-config was released

Reply:
Thank you for that info. I knew it was around that version but wasn't sure off the top of my head. I will generate some configs with 6.4.0 and 6.5.0 just to confirm that is the cut off and then update this.

Reply:
Updated to 6.4.0. 6.4.0 includes usernameAttributes and omits loginMechanisms. 6.3.0 was the last version to use `loginMechanisms`.